app/questionnaire/questionnaire_store_updater.py

```diff
@@ -16,15 +19,15 @@
 class QuestionnaireStoreUpdater:
     """Component responsible for any actions that need to happen as a result of updating the questionnaire_store"""
 
-    EMPTY_ANSWER_VALUES: Tuple = (None, [], "", {})
+    EMPTY_ANSWER_VALUES: tuple = (None, [], "", {})
```

Comment:
Tuple of?

Reply:
Since the type will be inferred I've dropped this hint

Reply:
Still outstanding?

Reply:
outdated, should be ok to resolve now